src/Views/ProgressView.vala

```diff
@@ -183,6 +183,20 @@ public class ProgressView : AbstractInstallerView {
             value = 0
         };
 
+        // Sectors may also be constructed using different units of measurements, such as
+        // by megabytes and percents. The library author can choose whichever unit makes
+        // more sense for their use cases.
+        var boot_sector = Distinst.Sector () {
+            flag = Distinst.SectorKind.MEGABYTE,
+            value = 512
+        };
+
+        // Sectors may also use the end of the disk
+        var swap_sector = Distinst.Sector () {
+            flag = Distinst.SectorKind.MEGABYTE_FROM_END,
+            value = 4096
```

Comment:
I've heard of smart algorithms setting at least the size of the RAM to allow deep sleep
But I'm also pretty sure that I've heard that there is no need for a swap partition nowadays and that it's possible to use a swap file :man_shrugging: